lgc/include/lgc/state/ResourceUsage.h

```diff
@@ -286,7 +286,7 @@ struct ResourceUsage {
 
       struct {
         uint64_t u64All;
-      } perStage;
+      } perStage = {};
```

Comment:
I don't think you need these two new initializers, as they are inside builtInUsage, which already has an initializer, right?

Maybe that's causing your test failures, but if so I don't understand why.

Reply:
> Maybe that's causing your test failures, but if so I don't understand why.

Yes, that is really strange. By doing so, the clang build could pass. I will retest initial change. But from the test report, it seems some fields are not correctly zeroed. I couldn't reproduce it on my local machine.

Reply:
I retest it. The result is as what I said. Testing from clang build fails. I am confused as well.